src/components/ScoreDetails/ScoreDetails.css

```diff
@@ -0,0 +1,3 @@
+.toggleSeparator {
```

Comment:
Need to add a `.ScoreDetails` namespace before this selector and another one in `ScoreDetails.js` `Container` element.